services/user.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
The `User` model is imported but not used directly in the code. Consider removing this import if it's not necessary.